lib/handlers/Chrome55.js

```diff
@@ -20,7 +20,7 @@ class Handler extends EnhancedEventEmitter
 		this._pc = new RTCPeerConnection(
 			{
 				iceServers         : settings.turnServers || [],
-				iceTransportPolicy : 'all',
+				iceTransportPolicy : settings.iceTransportPolicy || 'all',
```

Comment:
Let's remove the `|| 'all'` here and in all the handlers. It's not needed since `this._settings` in `Room.js` already has it as default value.